cmd/peerswap-plugin/main.go

```diff
@@ -107,6 +108,22 @@ func run() error {
 		return err
 	}
 
+	// We want to make sure that cln is synced and ready to use before we
+	// continue to start services.
+	log.Infof("Waiting for cln to be synced")
+	err = utils.WaitFor(func() bool {
+		info, err := lightningPlugin.GetLightningRpc().GetInfo()
+		if err != nil {
+			log.Infof("rpc.GetInfo() %v", err)
+			return false
+		}
+
+		return info.IsBitcoindSync() && info.IsLightningdSync()
+	}, 10*time.Second, 10*time.Minute)
```

Comment:
```
2022-07-13T16:54:45.513Z INFO    plugin-peerswap-plugin: PeerSwap Initialized, running PeerSwap commit 95eac6a5281fb3cfbae549222ebd1aa29485f0b4
2022-07-13T16:54:45.555Z INFO    plugin-peerswap-plugin: Waiting for cln to be synced
2022-07-13T17:04:48.800Z INFO    plugin-peerswap-plugin: Killing plugin: exited during normal operation
```

We found 10 minutes can be too short. Perhaps it should wait forever? It is CLN's problem to let us know when it thinks it is ready.